src/main/java/frc/robot/subsystems/vision/CameraPoses.java

```diff
@@ -5,33 +5,27 @@
 import edu.wpi.first.math.geometry.Translation3d;
 import edu.wpi.first.math.util.Units;
 
-/** Configuration class containing the poses of all cameras on the robot for vision processing. */
 public class CameraPoses {
-
-  private CameraPoses() {
-    throw new IllegalStateException("Utility class");
-  }
-
```

Comment:
I wouldn't touch this -- did it cause some issue? This is really just a protection against instantiating the class, so is that happening somewhere?